deployment/data-streams/changeset/fee-manager/test_helpers.go

```diff
@@ -33,32 +34,20 @@ func NewDefaultOptions() DataStreamsTestEnvOptions {
 	}
 }
 
-func NewDataStreamsEnvironment(t *testing.T, opts DataStreamsTestEnvOptions) (DataStreamsTestEnvSetupOutput, error) {
+func DeployTestEnvironment(t *testing.T, opts DataStreamsTestEnvOptions) (DataStreamsTestEnvSetupOutput, error) {
 	t.Helper()
 
-	e := testutil.NewMemoryEnv(t, opts.DeployMCMS, 0)
+	testEnv := testutil.NewMemoryEnvV2(t, testutil.MemoryEnvConfig{
+		ShouldDeployMCMS:      opts.DeployMCMS,
+		ShouldDeployLinkToken: opts.DeployLinkToken,
+	})
```

Comment:
The options here are a little limited - we sometimes need to specify number of oracle/bootstrap nodes as well labels to attach to them. It's ok to have a limited set here - we can add a comment saying that if you need something more detailed you should make the calls yourself. Or we can add some optional fields.

Reply:
These already exist as options though in the MemoryEnvConfig